src/pybind/mgr/dashboard/frontend/src/app/shared/constants/cephfs.constant.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_SUBVOLUME_GROUP = '_nogroup';
```

Comment:
nit: missing new line
```suggestion
export const DEFAULT_SUBVOLUME_GROUP = '_nogroup';

```